package.json

```diff
@@ -22,7 +22,6 @@
     "lint": "resin-lint lib tests",
     "mocha": "mocha --compilers coffee:coffee-script/register tests/**/*.spec.coffee",
     "test": "npm run build && npm run lint && npm run mocha",
-    "prepublish": "require-npm4-to-publish",
     "prepublishOnly": "npm test"
```

Comment:
You should be able to remove this as well

Reply:
You mean the `prepublishOnly`?